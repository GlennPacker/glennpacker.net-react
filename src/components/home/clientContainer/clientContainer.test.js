import React from 'react';
import ClientContainer from './index';
import { shallow } from 'enzyme';


describe('Client Container', () => {
    it('renders correctly', () => {
        const clients = ['client A', 'client B'];
        const actions = { loadClients: jest.fn() }

        const wrapper = shallow(
            <ClientContainer
                clients={clients}
                actions={actions}
            />
        )

        expect(wrapper).toMatchSnapshot();
    })
})