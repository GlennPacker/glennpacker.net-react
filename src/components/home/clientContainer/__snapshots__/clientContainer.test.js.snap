// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Client Container renders correctly 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
